actors/builtin/storage_miner/storage_miner_actor_state.go

```diff
@@ -56,23 +60,18 @@ func (mps *MinerPoStState) Is_DetectedFault() bool {
 	return mps.NumConsecutiveFailures > 0
 }
 
-type SectorState int64
-
-const (
-	PreCommit SectorState = iota
-	Active
-	TemporaryFault
-)
+type SectorPreCommitOnChainInfo struct {
+	Info SectorPreCommitInfo
+	PreCommitDeposit abi.TokenAmount
+	PreCommitEpoch abi.ChainEpoch
+}
 
 type SectorOnChainInfo struct {
-	State                 SectorState
 	Info                  SectorPreCommitInfo // Also contains Expiration field.
-	PreCommitDeposit      abi.TokenAmount
-	PreCommitEpoch        abi.ChainEpoch
-	ActivationEpoch       abi.ChainEpoch // -1 if still in PreCommit state.
+	ActivationEpoch       abi.ChainEpoch
```

Comment:
Please document this and all fields. E.g. `The epoch at which the sector commitment was proven`.